### PR TITLE
[#5172]fix bug:Iceberg catalog audit user is not correct

### PR DESCRIPTION
**Pull request by lsyulong**
### What changes were proposed in this pull request?

[#5172]fix bug:Iceberg catalog audit user is not correct

### Why are the changes needed?

Fix: #5172 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

No need

### CODE REVIEW

**Comment by jerryshao** [comment]
@FANNG1 please help to review this.

File: catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java [hunk 1]
```diff
@@ -589,6 +590,6 @@ public void testConnection(
 
   // TODO. We should figure out a better way to get the current user from servlet container.
```
Comment (FANNG1) [inline catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java:591]: could you remove this todo?

Comment (lsyulong) [reply catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java:591]: No, I'll check my code

Comment (lsyulong) [reply catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java:591]: ok

**Comment by FANNG1** [comment]
@lsyulong, thanks for your PR. There are similar problems in `JdbcCatalog`. Would you like to fix them together in this PR?
https://github.com/apache/gravitino/blob/main/catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/JdbcCatalogOperations.java#L573-L576

**Comment by lsyulong** [comment]
> @lsyulong, thanks for your PR. There are similar problems in `JdbcCatalog`. Would you like to fix them together in this PR? https://github.com/apache/gravitino/blob/main/catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/JdbcCatalogOperations.java#L573-L576

ok,thank you

**Review by FANNG1 — APPROVED** [review]

**Comment by FANNG1** [comment]
@lsyulong  thanks for your work!

**Pull request merged**